test/unit/annotation_spec.js

```diff
@@ -233,6 +233,226 @@ describe('annotation', function() {
     });
   });
 
+  describe('MarkupAnnotation', function () {
```

Comment:
In my previous PR I already added a `describe` block for this (see https://github.com/mozilla/pdf.js/blob/master/test/unit/annotation_spec.js#L320), so you can simply place your new tests in there so we don't have to create the block here again.

Reply:
Didn't see that :). Thank you. Moved.